README.md

```diff
@@ -440,21 +440,30 @@ The instance of this Reporter (.asset file) can have the following settings.
 </dl>
 
 
-### Slack Reporter
+### SlackReporter
 
 A Reporter that post report to Slack.
 
 The instance of this Reporter (.asset file) can have the following settings.
 
 <dl>
-  <dt>Slack Token</dt><dd>Web API token used for Slack notifications. If omitted, no notifications will be sent.
+  <dt>Slack Token</dt><dd>Oath token of Slack Bot used for notifications. If omitted, no notifications will be sent.
```

Comment:
I think thai it is OAuth token

Reply:
Thanks! fixed in f45f0226706a5483a3bd1217ac830a725b841bd0